grok.go

```diff
@@ -129,7 +129,10 @@ func (g *Grok) addPatternsFromMap(m map[string]string) error {
 	patternDeps := graph{}
 	for k, v := range m {
 		keys := []string{}
-		for _, key := range canonical.FindAllStringSubmatch(v, -1) {
+		for _, key := range normal.FindAllStringSubmatch(v, -1) {
```

Comment:
Shouldn't this range be using 

**valid.**FindAllStringSubmatch

instead of 

**normal.**FindAllStringSubmatch